src/systemtest/resources/features/ArtifactFlow.feature

```diff
@@ -7,7 +7,7 @@ Feature: Artifact system test
     And some subscriptions are set up
     And a jenkins job '"ArtC2Job"' from '"src/systemtest/resources/JenkinsShellScripts/ArtC2Script.txt"' is created
     And a jenkins job '"TCTJob"' from '"src/systemtest/resources/JenkinsShellScripts/TCTScript.txt"' is created
-    And a jenkins job '"TCS&TCTJob"' from '"src/systemtest/resources/JenkinsShellScripts/TCS&TCTScript.txt"' is created
+    And a jenkins job '"TCSTCFJob"' from '"src/systemtest/resources/JenkinsShellScripts/TCSTCFScript.txt"' is created
```

Comment:
Add a step that will remove the jobs and the subscriptions.

Reply:
Done